docs/sources/reference/api/docker_remote_api_v1.19.md

```diff
@@ -220,12 +221,15 @@ Json Parameters:
       container
 -   **ExposedPorts** - An object mapping ports to an empty object in the form of:
       `"ExposedPorts": { "<port>/<tcp|udp>: {}" }`
+-   **VolumeDriver** - A string value containing the volume driver to use, `local` by default.
```

Comment:
@lukemarsden I guess it would be good to have something there, we set the driver to `local` when it's empty, so people don't necessarily need to set it themselves.